sdk/synapse/Azure.Analytics.Synapse.Artifacts/src/Generated/Models/NotebookParameter.Serialization.cs

```diff
@@ -21,15 +21,8 @@ void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
             writer.WriteStartObject();
             if (Optional.IsDefined(Value))
             {
-                if (Value != null)
```

Comment:
Property is not nullable in spec, however it is checked via `Optional.IsDefined`